README.md

```diff
@@ -21,10 +21,57 @@ other components of OpenShift via support for a baremetal platform type.
 
 # Instructions
 
-## Configuration
+## Preparation
+
+Considering that this is a new install on a clean OS, the next tasks should be performed prior the installation:
+
+1. Enable passwordless sudo for the current user
+
+    `echo "$USER  ALL=(ALL) NOPASSWD: ALL" >/etc/sudoers.d/${USER}`
+
+2. In case of RHEL, invoke `subscription-manager` in order to `register` and `attach` the subscription
+
+3. Install new packages
 
-Make a copy of the `config_example.sh` to `config_$USER.sh`, and set the
-`PULL_SECRET` variable to the secret obtained from cloud.openshift.com.
+    `sudo dnf upgrade -y`
+    `sudo dnf install -y git make wget jq`
+
+4. Clone the dev-scripts repository
+
+    `git clone https://github.com/openshift-metal3/dev-scripts`
+
+5. Create a config file
+
+    `cp config_example.sh config_$USER.sh`
+
+6. Obtain secrets:
```

Comment:
Here I feel the lack of a little bit of description: it could not be obvious, for a new user, that the following steps are meant to generate a combined secret composed by a personal secrete merged with a shared CI secret. The following points are an implementation of this, but I'd prefer to introduce the reader with a small description first